lang/en.json

```diff
@@ -912,6 +912,9 @@
         "SPAFollowStep14": "Add a version name, for example %1$s1.0%2$s",
         "SPAFollowStep15": "Click the green \"Create New Version And Publish Release\" button.",
         "SPAFollowStep16": "Copy the code below and paste it in the header of your SPA / PWA. We recommend pasting it immediately before the closing %1$s tag. %2$sLearn more%3$s.",
-        "ReactFollowStep16": "Inject the %1$sMatomo Tag Manager JS%2$s code into your %3$s (or other relevant files), we recommend doing this using the \"%4$s\" method. The example below shows how to add the %1$sMatomo Tag Manager JS%2$s code into a \"%5$s\" app in %6$s."
+        "ReactFollowStep16": "Inject the %1$sMatomo Tag Manager JS%2$s code into your %3$s (or other relevant files), we recommend doing this using the \"%4$s\" method. The example below shows how to add the %1$sMatomo Tag Manager JS%2$s code into a \"%5$s\" app in %6$s.",
+        "SiteWithoutDataMtmIntro": "The Matomo Tag Manager may be a better choice than tracking via the regular JavaScript code. It lets you manage and unify all your tracking and marketing tags (also known as “pixels” or custom HTML/JS tags) in one place. Using the Matomo Tag Manager makes your life easier when you want to modify any of these tags, as you will no longer need a developer to make the needed changes for you. %1$sLearn more%2$s.",
+        "SiteWithoutDataMtmStep2": "If you would like to customize the Matomo tracking for this container, you can do so from the ",
```

Comment:
@snake14 can we rename this to `SiteWithoutDataMtmStep02` ? I remember @michalkleiner  adding this in Development channel to make it easy for him to translate later

Reply:
If it's a new step and likely there will be over 10 steps at some time, it would be preferred to keep two digits numbering but not a huge deal if not likely to get too long a list.

Reply:
Oh, yeah. I forgot about that. I don't think this will exceed 10 steps, so the sorting shouldn't be an issue.